packages/hmr-plugin/src/hmr-bootstrap.ts

```diff
@@ -0,0 +1,28 @@
+import { ApplicationRef, NgModuleRef } from '@angular/core';
+import { createNewHosts } from '@angularclass/hmr';
+
+import { hmrBeforeOnDestroy, hmrDoBootstrap, setStateInHmrStorage, validateExistHmrStorage } from './internal';
+import { HmrBootstrapFn, NgxsStoreSnapshot } from './symbols';
+
+export const hmrNgxsBootstrap: HmrBootstrapFn = (module, bootstrap, autoClearLogs = true) => {
+  let ngModule: NgModuleRef<any>;
+  validateExistHmrStorage();
+  module.hot.accept();
+
+  bootstrap().then((ref: NgModuleRef<any>) => (ngModule = hmrDoBootstrap(ref)));
+
+  module.hot.dispose(() => {
+    if (autoClearLogs) {
+      console.clear();
+      console.log('[NGXS HMR] clear old logs...');
+    }
+
+    const snapshot: Partial<NgxsStoreSnapshot> = hmrBeforeOnDestroy<NgxsStoreSnapshot>(ngModule);
+    setStateInHmrStorage(snapshot);
```

Comment:
Small refactor...
These 2 lines could also be moved to the './internal' file as a `hmrDoDispose` method (or some other similar name).
Then the logic about the storage can remain encapsulated in the one file.
Potentially even the `validateExistingHmrStorage` could be called from within './internal' too (maybe in the bootstrap/dispose).

Reply:
Fixed in my recent commit to this branch.